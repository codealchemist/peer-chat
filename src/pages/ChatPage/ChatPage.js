import React from 'react'
import Chat from 'components/Chat'
import Header from 'components/Header'

const ChatPage = () => (
  <>
    <Header type="subtleRight" />
    <Chat />
  </>
)

export default ChatPage
