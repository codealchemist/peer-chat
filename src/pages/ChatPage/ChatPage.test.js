import React from 'react'
import { mount } from 'enzyme'
import ChatPage from './ChatPage'

jest.mock('components/Chat', () => 'div')

describe('ChatPage', () => {
  it('should render', () => {
    const wrapper = mount(<ChatPage />)
    expect(wrapper).toMatchSnapshot()
  })
})
