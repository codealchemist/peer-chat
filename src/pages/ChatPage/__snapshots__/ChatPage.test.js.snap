// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatPage should render 1`] = `
<ChatPage>
  <Header
    type="subtleRight"
  >
    <styled.div
      type="subtleRight"
    >
      <StyledComponent
        forwardedComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bdVaJa",
              "isStatic": false,
              "lastClassName": "lepajG",
              "rules": Array [
                "
  h1 {
    font-family: 'Playfair Display', serif;
  }

  ",
                [Function],
                "
",
              ],
            },
            "displayName": "styled.div",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bdVaJa",
            "target": "div",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          }
        }
        forwardedRef={null}
        type="subtleRight"
      >
        <div
          className="sc-bdVaJa lepajG"
          type="subtleRight"
        >
          <h1>
            Chateer
          </h1>
        </div>
      </StyledComponent>
    </styled.div>
  </Header>
  <div />
</ChatPage>
`;
